backend/src/util/mapping/MappableObject.js

```diff
@@ -27,6 +27,13 @@ class MappableObjectForFront extends MappableObject {
 class MappableObjectForBack extends MappableObject {
   constructor(input, mappings) {
     super(input, mappings, 'front', 'back');
+
+    const keys = Object.keys(this.data);
+    for (let key of keys) {
+      if (key.includes('@OData')) {
```

Comment:
While the case isn't the same, **@odata.bind** is used to create entity relationships in CRM. To be safer we can match endsWith **@OData.Community.Display.V1.FormattedValue** which is only ever used for read-only values.

Reply:
Bah, good to know. Thanks!